en/orm/saving-data.rst

```diff
@@ -770,21 +774,20 @@ The ``save()`` method will return the modified entity on success, and ``false``
 on failure. You can disable rules and/or transactions using the
 ``$options`` argument for save::
 
-    // In a controller or table method.
+    // In a controller or table method
     $articles->save($article, ['checkRules' => false, 'atomic' => false]);
 
 Saving Associations
 -------------------
 
-When you are saving an entity, you can also elect to save some or all of the
-associated entities. By default all first level entities will be saved. For
-example saving an Article, will also automatically update any dirty entities
-that are directly related to articles table.
+When you are saving an entity, by default all associated entities,
+that are marekd dirty will be saved. You can however elect to save
+no associated entities by setting the ``associated`` option to ``false``.
```

Comment:
@lorenzo confirmed on slack that the manual is correct and the API docblock needs updating :)